cwltool/main.py

```diff
@@ -655,10 +655,8 @@ def loadref(b, u):
 def print_pack(document_loader, processobj, uri, metadata):
     # type: (Loader, Union[Dict[Text, Any], List[Dict[Text, Any]]], Text, Dict[Text, Any]) -> str
     packed = pack(document_loader, processobj, uri, metadata)
-    if len(packed["$graph"]) > 1:
-        return json.dumps(packed, indent=4)
-    else:
-        return json.dumps(packed["$graph"][0], indent=4)
+
+    return json.dumps(packed, indent=4)
```

Comment:
I don't think this conditional statement had any significance at this place. 
@mr-c Something I might be missing out?